asus/zephyrus/gu605my/default.nix

```diff
@@ -1,16 +1,29 @@
-{ ... }:
+{ lib, ... }:
 
 {
   imports = [
     ../../../common/cpu/intel
     ../../../common/gpu/nvidia/prime.nix
-    ../../../common/gpu/nvidia/ampere
+    ../../../common/gpu/nvidia/ada-lovelace
     ../../../common/pc/laptop
     ../../../common/pc/ssd
   ];
 
-  hardware.nvidia.prime = {
-    intelBusId = "PCI:0:2:0";
-    nvidiaBusId = "PCI:1:0:0";
+  hardware.nvidia = {
+    prime = {
+      intelBusId = "PCI:0:2:0";
+      nvidiaBusId = "PCI:1:0:0";
+    };
+
+    dynamicBoost.enable = lib.mkDefault true;
+  };
+
+  services = {
+    asusd.enable = lib.mkDefault true;
```

Comment:
Is this something we should include in all asus laptops?

Reply:
I saw it in another asus laptop an I think at least adding it as default does not hurt. Has functionality to control bios features matrix and slash screen and so forth. At least for all Zephyrus laptops it's a good addition.